Calculator/Program.cs

```diff
@@ -0,0 +1,191 @@
+                PrintResult(result);
+
+                DisplayCalculatorUsage(calculator);
+
+                DisplayMemory(calculator);
+
+                DisplayCleanupOptions();
+
+                endApp = ProcessCleanup(endApp, calculator);
+            }
+            calculator.Finish();
+            return;
+        }
+
+        private static void GetInputs(Calculator calculator, out double cleanNum1, out double cleanNum2)
+        {
+            Console.Write("Type a number, and then press Enter: ");
+            cleanNum1 = ValidateNumberInput(calculator);
+            Console.Write("Type another number, and then press Enter: ");
+            cleanNum2 = ValidateNumberInput(calculator);
+        }
+
+        private static bool ProcessCleanup(bool endApp, Calculator calculator)
+        {
+            string selection = Console.ReadLine();
+            switch (selection)
+            {
+                case "n":
+                    endApp = true;
+                    break;
+                case "c":
+                    calculator.ClearHistory();
+                    break;
+                default:
+                    break;
+            }
+            return endApp;
+        }
+
+        private static void DisplayCalculatorUsage(Calculator calculator)
+        {
+            Console.Write($"This calculator has been used {calculator.CalculationsCompleted} times.\n\n");
+        }
+
+        private static void PrintResult(double result)
+        {
+            if (double.IsNaN(result))
+            {
+                Console.WriteLine("This operation will result in a mathematical error.\n");
+            }
+            else Console.WriteLine("Your result: {0:0.##}\n", result);
+        }
+
+        private static double CalculateResult(Calculator calculator, double cleanNum1, double cleanNum2, string op)
+        {
+            double result = 0;
+            try
+            {
+                result = calculator.DoOperation(cleanNum1, cleanNum2, op);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine("Oh no! An exception occurred trying to do the math.\n - Details: " + e.Message);
+            }
+            return result;
+        }
+
+        private static void DisplayMemory(Calculator calculator)
+        {
+            Console.WriteLine("---------Memory--------\n");
+            string memory = string.Empty;
+            int i = 0;
+            foreach (Calculation calc in calculator.CalculationMemory)
+            {
+                memory += $"{i}: {calc}";
+                i++;
+            }
+            Console.WriteLine(memory);
+            Console.WriteLine("To reference memory in your input type 'M' followed by the memory reference (e.g. M2)\n");
+        }
+
+        private static void DisplayOperatorOptions()
+        {
+            Console.WriteLine("\nChoose an operator from the following list:\n");
```

Comment:
This is optional, but be aware of this possibility. Using @ prevents you from repeating Console.Writeline several times. I always find that less code is better: 
![image](https://user-images.githubusercontent.com/108711476/203435016-fd654dff-9773-4186-8d08-6108e80846ee.png)